CardTech.ResearchSite/CardTech.ResearchSite/Pages/About.cshtml

```diff
@@ -6,3 +6,29 @@
 
 <h1>About</h1>
 
+<body>
```

Comment:
I'm having trouble understanding this... is there a reason we have some sample code here?